buildpack.md

```diff
@@ -397,13 +395,15 @@ The purpose of build is to transform application source code into runnable artif
 During the build phase, typical buildpacks might:
 
 1. Read the Buildpack Plan in `<plan>` to determine what dependencies to provide.
-2. Provide the application with dependencies for launch in `<layers>/<layer>`.
-3. Provide subsequent buildpacks with dependencies in `<layers>/<layer>`.
-4. Compile the application source code into object code.
-5. Remove application source code that is not necessary for launch.
-6. Provide start command in `<layers>/launch.toml`.
-7. Write a partial Bill-of-Material to `<layers>/launch.toml` describing any provided application dependencies.
-8. Write a partial Bill-of-Material to `<layers>/build.toml` describing any provided build dependencies.
+1. Provide the application with dependencies for launch in `<layers>/<layer>`.
+1. Reuse application dependencies from a previous image by appending by `launch = true` to `<layers>/<layer>.toml`.
```

Comment:
```suggestion
1. Reuse application dependencies from a previous image by appending `launch = true` to `<layers>/<layer>.toml`.
```

Reply:
This also requires that the buildpack does NOT create a `<layers>/<layer>` of it's own... IDK if that is implied or would be worth spelling out. Personally I found myself re-reading this a couple of times...